__testfixtures__/imports/redirectExport.output.js

```diff
@@ -0,0 +1,8 @@
+import Sound as exports;
+
+import ..CellView as exports;
+
+import .InputEvent as exports.InputEvent;
```

Comment:
Wrong?  The mods should warn / bail on any of the `redirectExport` cases, meaning they will have to be updated by hand